.travis.yml

```diff
@@ -19,7 +19,7 @@ addons:
     update: true
     packages:
       - libxmlsec1-dev
-      - libgeoip-dev
+      - libmaxminddb-dev
```

Comment:
This isn't important, since it's just a speed improvement for maxminddb, but we at least have no need for libgeoip-dev anymore, so might as well swap it.